### PR TITLE
[Docs] Updates to V1.5.1 change log 

### PR DESCRIPTION
**Pull request by jillnogold**

### CODE REVIEW

**Review by gilad-mck — CHANGES_REQUESTED** [review]

File: docs/change-log/index.md [hunk 1]
```diff
@@ -734,26 +734,44 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
+|ML-4642|The UI can get stuck when query results are too large to display. |Add a filter (or narrow it) to retrieve fewer results.|                 v1.5.0|   
+|ML-4678|When tagging a specific version of a model using the SDK, it clears the tags from the rest of the versions.|Use the UI instead. | v1.5.0 |
+|ML-4698|Parameters that are passed to a workflow are limited to 10000 chars.| NA, external Kubeflow limitation. | v1.5.0 |
+|ML-4714|Logs got truncated in the MLRun UI logs page for jobs that have a high number of logs or run for over day.  | NA | v1.5.0 |
 |ML-4740|When running function `batch_inference_v2` from the SDK, the `ingest()` function accepts 3 parameters as Data-item or other types: `dataset`, `model_path` and `model_endpoint_sample_set`. If you provided these parameters as non Data-items and later on you want to rerun this function from the UI, you need to provide these parameters as Data-item.|Prepare suitable Data-item and provide it to the batch-rerun UI.| v1.5.0    |
+|ML-4753|When rebuilding a new image with a name that has already been used, the dependencies of the new image are not updated.  | NA | v1.5.0 |
```
Comment (alonmr) [inline docs/change-log/index.md:754]: I don't think thats a real bug - just need to change the image pull policy. let's remove it

File: docs/change-log/index.md [hunk 2]
```diff
@@ -734,26 +734,44 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
 |ML-3480|Documentation: request details on label parameter of feature set definition.| NA | v1.2.1    |
 |NA|The feature store does not support schema evolution and does not have schema enforcement.| NA| v1.2.1    |
+|ML-3521|Cannot schedule a workflow without a remote source. | NA| v1.2.)    |
 |ML-3640|When running a remote function/workflow, the `context` global parameter is not automatically injected.| Use `get_or_create_ctx`| v1.3.0    |
 |ML-3714|It can happen that an MLrun pod succeeds but there's an error in Kubeflow. | NA | v1.3.0 |
 |ML-3804|A serving step with no class does not inherit parameters from the function spec. |Create a class to forward the parameters. See [Create a single step](../serving/writing-custom-steps.html#create-a-single-step). | v1.3.1 |
+|ML-4107| On scheduled ingestion (storey engine) from CSV source, ingests all the source on each schedule iteration. | NA | v1.4.0 |
```
[the remark this answers is not in the document]
Comment (jillnogold) [reply docs/change-log/index.md]: yes, updated

File: docs/change-log/index.md [hunk 3]
```diff
@@ -734,26 +734,44 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
 |ML-4442|After a model is deployed without applying monitoring (`set_tracking()` was not set on the serving function), monitoring cannot be added.|Delete the existing model endpoint (`mlrun.get_run_db().delete_model_endpoint()`), then redeploy the model.| v1.5.0 |
 |ML-4585|The `mlrun/mlrun` image does not support mpijob. | Create your own image that includes mpijob.  | v1.5.0 |
 |ML-4613|UI: The Batch Inference Parameters has an incorrect hint on map type. The correct hint is "The 'dict' values should be in JSON key:value format, e.g. {"hello":"world"}" | NA | v1.5.0 |
+|ML-4617|Incorrect error message when using a feature vector as an input to a job without first calling `get_offline_features` on the vector. |Apply `get_offline_features()` on the feature vector and provide a target.| v1.5.0 |
+|ML-4642|The UI can get stuck when query results are too large to display. |Add a filter (or narrow it) to retrieve fewer results.|                 v1.5.0|   
+|ML-4678|When tagging a specific version of a model using the SDK, it clears the tags from the rest of the versions.|Use the UI instead. | v1.5.0 |
```
[the remark this answers is not in the document]
Comment (jillnogold) [reply docs/change-log/index.md]: updated

File: docs/change-log/index.md [hunk 4]
```diff
@@ -734,26 +734,44 @@ with a drill-down to view the steps and their details. [Tech Preview]
 |ML-3445|`project.deploy_function` operation might get stuck when running v1.3.0 demos on an Iguazio platform running v3.2.x.| Replace code: `serving_fn = mlrun.new_function("serving", image="python:3.9", kind="serving", requirements=["mlrun[complete]", "scikit-learn~=1.2.0"])` with: <br>`function = mlrun.new_function("serving", image="python:3.9", kind="serving") function.with_commands([ "python -m pip install --upgrade pip", "pip install 'mlrun[complete]' scikit-learn==1.1.2", ])`|v1.3.0    |
 |ML-3480|Documentation: request details on label parameter of feature set definition.| NA | v1.2.1    |
 |NA|The feature store does not support schema evolution and does not have schema enforcement.| NA| v1.2.1    |
+|ML-3521|Cannot schedule a workflow without a remote source. | NA| v1.2.)    |
```
[the remark this answers is not in the document]
Comment (jillnogold) [reply docs/change-log/index.md]: fixed

File: docs/change-log/index.md (hunk 1 above)
Comment (jillnogold) [reply docs/change-log/index.md]: removed

**Review by gilad-shaham — APPROVED** [review]

**Pull request merged**